VariantEncoding.md

```diff
@@ -39,13 +39,31 @@ Another motivation for the representation is that (aside from metadata) each nes
 For example, in a Variant containing an Array of Variant values, the representation of an inner Variant value, when paired with the metadata of the full variant, is itself a valid Variant.
 
 This document describes the Variant Binary Encoding scheme.
-[VariantShredding.md](VariantShredding.md) describes the details of the Variant shredding scheme.
+The [Variant Shredding spec](VariantShredding.md) describes the details of shredding Variant values as typed Parquet columns.
+
+## Variant in Parquet
 
-# Variant in Parquet
 A Variant value in Parquet is represented by a group with 2 fields, named `value` and `metadata`.
-Both fields `value` and `metadata` are of type `binary`, and cannot be `null`.
+The Variant group must be annotated with the `VARIANT` logical type.
+Both fields `value` and `metadata` are of type `binary`.
+The `metadata` field is required and must be a valid Variant metadata, as defined below.
+The `value` field is optional.
+When present, the `value` field must be a valid Variant value, as defined below. 
+The `value` field may be null only when parts of the Variant value are shredded according to the Variant Shreedding spec.
+
+This is the expected representation in Parquet:
 
-# Metadata encoding
+```
+optional group variant_event (VARIANT) {
+  required binary metadata;
+  optional binary value;
```

Comment:
Would an engine be violating the spec if it is not shredding and hence declaring the value field as required?

Reply:
No. The encoding spec says that the value must be required when not shredding. I didn't include that case here since this is documenting how to shred variants.

Reply:
Sorry, I forgot I was updating the encoding spec here. I'm updating this to match the `LogicalTypes.md` PR, which uses `required` for non-shredded Variant.